lib/stores/filesystem.js

```diff
@@ -245,6 +246,22 @@ class FilesystemStore {
     }
   }
 
+  async postObject(object) {
```

Comment:
This function isn't needed, you can just use the store's `putObject` implementation. S3's POST endpoint is essentially just an HTML form-friendly PUT endpoint.

Reply:
I'm not sure how true it is, I'm not a nodejs developer so I may be wrong,  but the big difference is that in my  case the file is already in the filesystem, so here we merely move it + hash it from FS, while the put implementation take the data from the response's body, so I'm a bit clueless on how to torture the controller to make it works with the existing filesyste/putObject

Reply:
OK I think I see - when it's defined, `object.content` is expected to be a stream. I was under the impression multiparty would expose streams for any files it discovered.

I think you should be able to call `fs.createReadStream(files['file'][0].path)` when you construct the `S3Object` in the controller.